### PR TITLE
[chrony receiver] Fixing flake

### PR DESCRIPTION
**Pull request by MovieStoreGuy**
**Description:**

Addressing bug in tests

**Link to tracking Issue:** 

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16304

**Testing:** 

I've run this locally but I suspect this is likely to happen more windows machines

**Documentation:** 

N/A

### CODE REVIEW

File: .chloggen/msg_fix-chrony-flake.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```
Comment (MovieStoreGuy) [inline .chloggen/msg_fix-chrony-flake.yaml:1]: I suspect the change log isn't needed for this, but added to pass the CI

**Review by codeboten — APPROVED** [review]
I'll add the skip changelog label, please remove the changelog file

**Comment by MovieStoreGuy** [comment]
@codeboten <3

I true hero of the CI!

**Review by jamesmoessis — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
@MovieStoreGuy thanks for the quick fix

**Pull request merged**